ReactiveCocoa/UIKit/tvOS/BindingTargetSubstituting.generated.swift

```diff
@@ -0,0 +1,160 @@
+// This is auto-generated by gyb.
+
+import Foundation
+import ReactiveSwift
+import enum Result.NoError
+import UIKit
+
+extension UIActivityIndicatorView {
```

Comment:
I don't think we should duplicate shared operators. They should be in a shared source file.